web-app/src/components/BookingModal/BookingModalForm/index.js

```diff
@@ -153,10 +169,11 @@ BookingModalForm.propTypes = {
   formIsValid: PT.bool.isRequired,
   createEvent: PT.func.isRequired,
   updateEvent: PT.func.isRequired,
+  booking: PT.object,
 };
 
 BookingModalForm.defaultProps = {
   formIsValid: true,
 };
 
-export default container(BookingModalForm);
+export default container(BookingModalForm);
```

Comment:
Not a big issue but its seems like your Prettier still isn't adding newlines at the end of the file.